### PR TITLE
Add note that CameraParameters necessity.

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: Examples/src/main/java/com/mapbox/vision/examples/ExternalVideoSourceActivity.java [hunk 1]
```diff
@@ -195,6 +201,15 @@ private void startFileVideoSource() {
                         ImageFormat.RGBA,
                         imageSize
                 );
+                // set CameraParameters them each time they change
```
Comment (yuryybk) [inline Examples/src/main/java/com/mapbox/vision/examples/ExternalVideoSourceActivity.java:204]: I'd add the comment exactly here ("...that this features still won't work for this particular example...")

File: Examples/src/main/kotlin/com/mapbox/vision/examples/ExternalVideoSourceActivityKt.kt [hunk 2]
```diff
@@ -170,6 +176,15 @@ class ExternalVideoSourceActivityKt : BaseActivity() {
                     ImageFormat.RGBA,
                     imageSize
                 )
+                // set CameraParameters them each time they change
```
Comment (yuryybk) [inline Examples/src/main/kotlin/com/mapbox/vision/examples/ExternalVideoSourceActivityKt.kt:179]: I'd add the comment exactly here ("...that this features still won't work for this particular example...")

File: Examples/src/main/java/com/mapbox/vision/examples/ExternalVideoSourceActivity.java (hunk 1 above)
Comment (yunikkk) [reply Examples/src/main/java/com/mapbox/vision/examples/ExternalVideoSourceActivity.java:204]: Well, ok

File: Examples/src/main/kotlin/com/mapbox/vision/examples/ExternalVideoSourceActivityKt.kt (hunk 2 above)
Comment (yunikkk) [reply Examples/src/main/kotlin/com/mapbox/vision/examples/ExternalVideoSourceActivityKt.kt:179]: Well, ok

**Pull request merged**